src/api-docs/source/includes/_wallet_rpc_auctions.md

```diff
@@ -1334,6 +1341,998 @@ address | Required | address submitting the bid
 value | Required | value of the bid (in HNS)
 
 
+## createopen
+
+```shell--vars
+name='possibility'
+```
+
+```shell--cli
+hsw-cli rpc createopen $name
+```
+
+```javascript
+const {WalletClient} = require('hs-client');
+const {Network} = require('hsd');
+const network = Network.get('regtest');
+
+const clientOptions = {
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const client = new WalletClient(clientOptions);
+
+(async () => {
+  const result = await client.execute('createopen', [ name ]);
+  console.log(result);
+})();
+```
+
+> createopen returns JSON structured like this: 
+
+```json
+{
+  "hash": "05ba8ab476f41213613813ebc63a14d7154857efc9a19b7e181852ab19c05e82",
+  "witnessHash": "6afe4493d6b68d855db9685380ea6316a9d4b48212e584cbf383dccf1acbf94b",
+  "mtime": 1537401957,
+  "version": 0,
+  "inputs": [
+    {
+      "prevout": {
+        "hash": "837b802fbad06cab6c8f789c4745cc56d1498a651a04973c8b3db310d90fec42",
+        "index": 0
+      },
+      "witness": [
+        "cc603c24fd90881b00899751d634fad8cfc67ac1289de2475f5c09117db3037335eb9983d38113be4d7c1895514f7d0ff411d2e72dc3ebb444e811146958ebc601",
+        "03b9843c1c40c210790e55052c3e8c56b49d2f0f1d00e8bdeb0237f076a128365a"
+      ],
+      "sequence": 4294967295,
+      "address": "rs1q9jd8fgq8xa0drggyd6azggpeslacdzx5gr04ss"
+    }
+  ],
+  "outputs": [
+    {
+      "value": 0,
+      "address": "rs1qkk62444euknkya4qws9cg3ej3au24n635n4qye",
+      "covenant": {
+        "type": 2,
+        "action": "OPEN",
+        "items": [
+          "01c05e8ea3d1c347342ef11c50fe5a1f621c942f7f8f7e0ee329eb883f93f9eb",
+          "00000000",
+          "706f73736962696c697479"
+        ]
+      }
+    },
+    {
+      "value": 999996200,
+      "address": "rs1qew8r8c2c74zpmkhwmxu6jkrhwmgegl5h6cfaz3",
+      "covenant": {
+        "type": 0,
+        "action": "NONE",
+        "items": []
+      }
+    }
+  ],
+  "locktime": 0,
+  "hex": "0000000001837b802fbad06cab6c8f789c4745cc56d1498a651a04973c8b3db310d90fec4200000000ffffffff0200000000000000000014b5b4aad6b9e5a76276a0740b8447328f78aacf5102032001c05e8ea3d1c347342ef11c50fe5a1f621c942f7f8f7e0ee329eb883f93f9eb04000000000b706f73736962696c69747928bb9a3b000000000014cb8e33e158f5441ddaeed9b9a9587776d1947e970000000000000241cc603c24fd90881b00899751d634fad8cfc67ac1289de2475f5c09117db3037335eb9983d38113be4d7c1895514f7d0ff411d2e72dc3ebb444e811146958ebc6012103b9843c1c40c210790e55052c3e8c56b49d2f0f1d00e8bdeb0237f076a128365a"
+}
+```
+Creates `OPEN` transaction without signing or broadcasting it.
+
+### Params
+Name | Default |  Description
+--------- | --------- | ---------
+name | Required | name to open bidding on
```

Comment:
All of these `create____` RPCs also take an optional "account" parameter.
`createopen` also accepts a totally unsused "force" bool which we have to include as a placeholder the same way "label" is required on a bunch of rpc calls even though we don't use it (but inherited that from bcoin which inherited that from Bitcoin Core...)

```
--> hsw-rpc help createopen
createopen "name" ( force "account" )
--> hsw-rpc help createbid
createbid "name" bid value ( "account" )
--> hsw-rpc help createreveal
createreveal "name" ( "account" )
--> hsw-rpc help createredeem
createredeem "name" ( "account" )
--> hsw-rpc help createupdate
createupdate "name" "data" ( "account" )
--> hsw-rpc help createrenewal
createrenewal "name" ( "account" )
--> hsw-rpc help createtransfer
createtransfer "name" "address" ( "account" )
--> hsw-rpc help createfinalize
createfinalize "name" ( "account" )
--> hsw-rpc help createcancel
createcancel "name" ( "account" )
--> hsw-rpc help createrevoke
createrevoke "name" ( "account" )
```

Reply:
Added in the description, however not using in examples for sake of simplicity, only example of `force account` for createopen is shown.